tests/data_utils.py

```diff
@@ -65,3 +80,83 @@ def make_tiny_ft_dataset(
         for sample in samples:
             _f.write(json.dumps(sample))
             _f.write('\n')
+
+
+def create_c4_dataset_xsmall(path: pathlib.Path) -> str:
+    """Creates a small mocked version of the C4 dataset."""
+    c4_dir = os.path.join(path, f'my-copy-c4')
+    downloaded_split = 'val_xsmall'  # very fast to convert
```

Comment:
I changed this to `val_xxsmall` in a different PR. Can you keep that change please? And while you're here, update the function name to also say xxsmall. Thanks!